data_steward/cdr_cleaner/manual_cleaning_rules/remove_operational_pii_fields.py

```diff
@@ -0,0 +1,101 @@
+# coding=utf-8
+"""
+Some new operational fields exists, that were not blacklisted in the RDR export. These rows needs to be dropped in the 
+RDR load process so they do not make it to CDR. These do not have concept_id maps. 
+The supplemental operational_pii_fields.csv shows all present PPI codes without a mapped concepts,
+indicating which should be dropped in the “drop_value” column
+"""
+import logging
+
+import bq_utils
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+
+LOGGER = logging.getLogger(__name__)
+
+OPERATIONAL_PII_FIELDS_TABLE = 'operational_pii_fields'
+INTERMEDIARY_TABLE = 'operation_pii_fields_to_delete'
+
+OPERATION_PII_FIELDS_INTERMEDIARY_QUERY = """
+CREATE OR REPLACE TABLE
+    `{project}.{dataset}_sandbox.{intermediary_table}` AS (
+  SELECT
+    *
+  FROM
+    {dataset}.observation
+  WHERE
+    observation_id IN (
+    SELECT
+      observation_id
+    FROM
+      `{project}.{dataset}_sandbox.{pii_fields_table}` as pii
```

Comment:
This line hard codes the `_sandbox` dataset naming convention.  This is repeated on line 48.  Consider changing `get_remove_operational_pii_fields_query(project_id, dataset_id)` to `get_remove_operational_pii_fields_query(project_id, dataset_id, sandbox_dataset_id)`